### DISCUSSION TITLE
Unable to connect to this room, please try again later.

### DISCUSSION THREAD

**Discussion by EthanReeve** [mozilla/hubs#4686] [Hubs Cloud Questions] [answered]
Hi there. I have successfully setup a personal version of the cloud without any errors, but I got this error while I was trying to create a room for the first time.
Please help.
<img width="1439" alt="image" src="https://user-images.githubusercontent.com/90125635/135234559-ddfafbec-a8ec-40b4-a09a-9a9401f6e43c.png">

**Comment by botissues** [comment]
I am having the same issue

**Comment by Savokr** [comment]
I am having the same issue with my Hubs cloud instance

**Reply by steph-rs** [reply]
+

**Comment by pattersonbl2** [comment]
are you still able to ssh into the instance and run a journalctl command ?

**Reply by EthanReeve** [reply]
Yep, I found all services were active.

**Comment by Valerian-denis** [comment]
I'm having the same issue: unable to connect to rooms on 1.1.2. 
This happens both for freshly installed instances or the ones I updated from 1.1.1 to 1.1.2.
/admin, /spoke and scene publishing are working fine, though.

**Comment by pattersonbl2** [comment]
Everyone should be able to update their stack using this guide https://hubs.mozilla.com/docs/hubs-cloud-aws-updating-the-stack.html#-upgrade-your-hubs-cloud-template-to-the-latest.  This will prevent port 80 from being used causing these issues.

**Reply by MarcusMorba** [reply]
I just installed the latest version on AWS, so this does not work for me.

**Reply by MarcusMorba** [reply]
Downgrade to 1.1.1 worked for me.

**Reply by pattersonbl2** [reply]
Thanks you for this work around. We found a bug that causing this issue. I will update when the bug is fixed

**Reply by pattersonbl2** [reply]
would it be possible for you to send the template you used to downgrade to 1.1.1  ?

**Reply by hnu-janfiedler** [reply]
> 
> 
> would it be possible for you to send the template you used to downgrade to 1.1.1 ?

Had the same issue, downgrading using this template fixed it: https://s3.amazonaws.com/awsmp-fulfillment-cf-templates-prod/5f452d36-bfff-4c02-a417-5c663b64fd08.2f1a2c81-f50a-4980-b2f5-196c3e17b2b7.template

**Comment by hnu-janfiedler** [comment, marked as answer]
I just saw in the Discord chat that this has been resolved, so updating to the 1.1.2 or setting up a new stack should have this fixed now.
